sds_data_manager/stacks/lambda_stack.py

```diff
@@ -34,8 +33,6 @@ def __init__(
         ----------
         scope : Construct
             The scope of the construct.
-        sdd_id : str
```

Comment:
Needs a docstring for construct_id

Reply:
Thanks! I updated this and also updated all other scope/construct_id docstrings to be exactly the same text now since they were all different and the inconsistency bothered me.